### ISSUE TITLE
[Feature Request] “Toggle read status” swipe action

### ISSUE THREAD

**Issue by natewind** [ProtonMail/proton-mail-android#194]
Currently, “Mark as read” swipe action does nothing when the message is already read. It would be more useful if it were a toggle, so that you wouldn’t have to make an extra tap to mark something unread.

**Comment by kylekatarnls** [comment]
Indeed, same actually happen for Tash/Archive/Spam, only the Favorite action is a toggle for now. I think it should at least "block" when the action won't make a change.
